<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowIcon">
   <iconset resource="../../vlyc.qrc">
    <normaloff>:/vlyc/res/vlyc2.png</normaloff>:/vlyc/res/vlyc2.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="pageVideo">
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="VideoWidget" name="video" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>1</horstretch>
            <verstretch>3</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="pageLibrary">
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QListWidget" name="libraryList">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
            <horstretch>3</horstretch>
            <verstretch>5</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <item>
           <property name="text">
            <string>Playlist</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="0" column="1" rowspan="2">
         <widget class="QStackedWidget" name="libraryStack">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>2</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <widget class="QWidget" name="page_2">
           <property name="minimumSize">
            <size>
             <width>502</width>
             <height>449</height>
            </size>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QTreeView" name="treeView">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="contextMenuPolicy">
               <enum>Qt::CustomContextMenu</enum>
              </property>
              <property name="selectionMode">
               <enum>QAbstractItemView::ExtendedSelection</enum>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="page_3">
           <property name="minimumSize">
            <size>
             <width>502</width>
             <height>449</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>502</width>
             <height>449</height>
            </size>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="controls" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="leftMargin">
        <number>10</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>10</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="time_bar">
         <item>
          <widget class="SeekSlider" name="position">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="TimeLabel" name="timeLabel"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="control_bar">
         <item>
          <widget class="PlayButton" name="btn_play">
           <property name="minimumSize">
            <size>
             <width>38</width>
             <height>38</height>
            </size>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="btn_prev">
           <property name="icon">
            <iconset resource="../../../QtVlc/vlc/modules/gui/qt4/vlc.qrc">
             <normaloff>:/toolbar/previous_b</normaloff>:/toolbar/previous_b</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="btn_stop">
           <property name="icon">
            <iconset resource="../../../QtVlc/vlc/modules/gui/qt4/vlc.qrc">
             <normaloff>:/toolbar/stop_b</normaloff>:/toolbar/stop_b</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="btn_next">
           <property name="icon">
            <iconset resource="../../../QtVlc/vlc/modules/gui/qt4/vlc.qrc">
             <normaloff>:/toolbar/next_b</normaloff>:/toolbar/next_b</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="btn_library">
           <property name="icon">
            <iconset resource="../../../QtVlc/vlc/modules/gui/qt4/vlc.qrc">
             <normaloff>:/toolbar/playlist</normaloff>:/toolbar/playlist</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="LoopButton" name="btn_repeat">
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QToolButton" name="btn_menu">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="icon">
            <iconset resource="../../../QtVlc/vlc/modules/gui/qt4/vlc.qrc">
             <normaloff>:/toolbar/dvd_menu</normaloff>:/toolbar/dvd_menu</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="btn_fullscreen">
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="../../../QtVlc/vlc/modules/gui/qt4/vlc.qrc">
             <normaloff>:/toolbar/fullscreen</normaloff>:/toolbar/fullscreen</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="subtitles">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="quality">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="SoundWidget" name="volume" native="true"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuOpen">
     <property name="title">
      <string>Open</string>
     </property>
     <addaction name="actionOpenFile"/>
     <addaction name="actionOpenFolder"/>
     <addaction name="actionOpenURL"/>
    </widget>
    <widget class="QMenu" name="menuRecent">
     <property name="title">
      <string>Recent</string>
     </property>
     <addaction name="separator"/>
     <addaction name="actionClearRecent"/>
    </widget>
    <addaction name="menuOpen"/>
    <addaction name="menuRecent"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuTools"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionBrowser">
   <property name="text">
    <string>Browser</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionOpenFile">
   <property name="text">
    <string>File</string>
   </property>
  </action>
  <action name="actionOpenURL">
   <property name="text">
    <string>URL</string>
   </property>
  </action>
  <action name="actionClearRecent">
   <property name="text">
    <string>Clear Recent Files</string>
   </property>
  </action>
  <action name="actionOpenFolder">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Folder</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TimeLabel</class>
   <extends>QLabel</extends>
   <header location="global">QtVlcWidgets/TimeLabel.h</header>
  </customwidget>
  <customwidget>
   <class>SeekSlider</class>
   <extends>QSlider</extends>
   <header location="global">QtVlcWidgets/SeekSlider.h</header>
  </customwidget>
  <customwidget>
   <class>SoundWidget</class>
   <extends>QWidget</extends>
   <header location="global">QtVlcWidgets/SoundWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PlayButton</class>
   <extends>QToolButton</extends>
   <header location="global">QtVlcWidgets/SpecialButtons.h</header>
  </customwidget>
  <customwidget>
   <class>VideoWidget</class>
   <extends>QWidget</extends>
   <header location="global">gui/videowidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>LoopButton</class>
   <extends>QToolButton</extends>
   <header location="global">QtVlcWidgets/SpecialButtons.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../vlyc.qrc"/>
  <include location="../../../QtVlc/vlc/modules/gui/qt4/vlc.qrc"/>
 </resources>
 <connections/>
</ui>
